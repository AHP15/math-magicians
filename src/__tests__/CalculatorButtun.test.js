import renderer from 'react-test-renderer';

import CalculatorButtun from '../components/CalculatorButton';

it('renders CalculatorButtun correctly', () => {
  const tree = renderer.create(<CalculatorButtun text="1" changeBg={false} handleClick={() => {}} />).toJSON();
  expect(tree).toMatchSnapshot();
});
