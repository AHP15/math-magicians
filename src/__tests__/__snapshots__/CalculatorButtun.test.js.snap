// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CalculatorButtun correctly 1`] = `
<button
  className=""
  onClick={[Function]}
  style={
    Object {
      "backgroundColor": "auto",
    }
  }
  type="button"
>
  1
</button>
`;
